cards/multilang-faq-accordion/component.js

```diff
@@ -68,11 +69,28 @@ class multilang_faq_accordionCardComponent extends BaseCard['multilang-faq-accor
 
     const contentEl = this._container.querySelector(accordionContentSelector);
     let isExpanded = this._container.querySelector(`.${accordionExpandedClass}`);
-    contentEl.style.height = `${isExpanded ? contentEl.scrollHeight : 0}px`;
+    let stayExpanded = this._container.querySelector(`.${stayExpandedClass}`);
+
+    const cardEl = this._container.querySelector(accordionCardSelector);
     const linkEls = contentEl.querySelectorAll('a');
+
+    if (stayExpanded && this._state._state.feedbackSubmitted) {
+      isExpanded = true;
+      cardEl.classList.toggle(accordionExpandedClass, isExpanded);
```

Comment:
does this work in ie11? you can test ie11 using browserstack (you can find a link to browserstack in your okta extension)

Reply:
I assume it should, since it is the same as in line 97 below. But I can test if there might still be issues

Reply:
it does not suppose second parameter for the toggle method for ie11 (https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle)

Reply:
Fixed